README.md

```diff
@@ -1,2 +1,106 @@
-# smslibrary
-SMS Library to send and work with SMS Messages between two users.
+# SMS library
+Android library to send and receive SMS messages quickly.
+
+# Installation
+Installation is done using [Jitpack](https://jitpack.io).
```

Comment:
We used jitpack when we were developing tbe killerapp to try and look from a total stranger point of view and it works like a charm, glad to see the central repo will be using it as well